### ISSUE TITLE
In ScriptCanvas: If EntitySpawnTicket is uncached into a variable, it should give an error message

### ISSUE THREAD

**Issue by AMZN-mnaumov** [o3de/o3de#9690]
If spawning something without caching EntitySpawnTicket, it will get automatically destroyed as soon as the ticket goes out of scope. This is how Spawnable API works and it's up to the user to maintain the lifecycle of the ticket. However it's not always obvious for someone new to the system.

A proposed solution is to track the workflow where ticket was not cached and display an error message when trying to save SC graph.

**Comment by monroegm** [comment]
Related to #8357

**Comment by AMZN-mnaumov** [comment]
currently blocked by https://github.com/o3de/o3de/issues/9895
